src/AzureClient/EntryPoint/EntryPoint.cs

```diff
@@ -115,9 +115,62 @@ private ArgumentValue GetArgumentValue(System.Reflection.ParameterInfo parameter
             {
                 return new ArgumentValue.Result(Newtonsoft.Json.JsonConvert.DeserializeObject<Result>(parameterValue)!);
             }
+            else if (parameterType.IsQArray())
+            {
+                var arrayType = parameterType.GenericTypeArguments.FirstOrDefault() ??
+                    throw new ArgumentException($"Could not get the type of array.");
+                if (arrayType == typeof(bool)) {
```

Comment:
Is there a way to consolidate this into a switch statement or expression?

Reply:
That would be ideal, but unfortunately since we make use of `typeof` we can't use a switch expression.